### ISSUE TITLE
Loss increases after training

### ISSUE THREAD

**Issue by ShellRedia** [MedicineToken/Medical-SAM2#19]
My model's performance actually decreased after training with a small amount of data. Any ideas for it?

**Comment by xukun-zhang** [comment]
Hello @ShellRedia,

I noticed your comment about the model's performance decreasing after training with a small amount of data. We encountered a related issue and are trying to reproduce the problem to better understand and resolve it.

Could you please provide more details on how you set up the experiment? Specifically, it would be helpful if you could share the following:

The software environment (e.g., Python version, PyTorch version, CUDA version)
The hardware configuration (e.g., GPU model)
Any specific parameters or settings you used during training
This information would be really valuable for us as we work on troubleshooting this issue. Thanks in advance for your assistance!

**Comment by ShellRedia** [comment]
Thank you for getting back to me. In fact, after further attempts yesterday, I found that using the method of fine-tuning with frozen weights （such as PEFT）can address this issue.
